distribution/docs/src/main/resources/_contents/_catalog-frameworks/catalog-api-contents.adoc

```diff
@@ -0,0 +1,247 @@
+
+==== Catalog API
+
+The Catalog API is an OSGi bundle (`catalog-core-api`) that contains the Java interfaces for the Catalog components and implementation classes for the Catalog Framework, Operations, and Data components.
+
+===== Catalog API Search Interfaces
+
+====== ${ddf-ui} Application Search Interface
+
+The ${branding} Search UI application provides a graphic interface to return results and locate them on an interactive globe or map.
+
+====== SSH Search Interface
+
+Additionally, it is possible to use a client script to remotely access ${branding} via SSH and send console commands to search and ingest data.
+
+===== Catalog Search Result Objects
+
+Data is returned from searches as Catalog Search Result objects.
+This is a subtype of Catalog Entry that also contains additional data based on what type of sort policy was applied to the search.
+Because it is a subtype of Catalog Entry, a Catalog Search Result has all Catalog Entry’s fields such as metadata, effective time, and modified time.
+It also contains some of the following fields, depending on type of search, that are populated by ${branding} when the search occurs:
+
+Distance:: Populated when a point radius spatial search occurs. Numerical value that indicates the result’s distance from the center point of the search.
+Units:: Populated when a point radius spatial search occurs. Indicates the units (kilometer, mile, etc.) for the distance field.
+Relevance:: Populated when a contextual search occurs. Numerical value that indicates how relevant the text in the result is to the text originally searched for.
+
+===== Search Programmatic Flow
+
+Searching the catalog involves three basic steps:
+
+. Define the search criteria (contextual, spatial, temporal, or compound – a combination of two or more types of searches).
```

Comment:
We should consider removing `compound` from our docs in references to queries. We can discuss more.